test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -30,6 +32,84 @@ private class InputData
             public float[] Features { get; set; }
         }
 
+        /// <summary>
+        /// Model Files: The (minimum) nuget version can be found in the model file.
```

Comment:
>model file [](start = 73, length = 10)

Is this an old model file or a current one? If it's current, shouldn't we create the model on the fly (similar to the other scenario below) instead of reading a static keep-model.zip file?